src/Umbraco.Abstractions/Configuration/IGlobalSettings.cs

```diff
@@ -63,14 +63,39 @@ public interface IGlobalSettings
         /// </summary>
         LocalTempStorage LocalTempStorageLocation { get; }
 
-        /// <summary>
-        /// Gets the location of temporary files.
-        /// </summary>
-        string LocalTempPath { get; }
-
         string UmbracoPath { get; }
         string UmbracoCssPath { get; }
         string UmbracoScriptsPath { get; }
         string UmbracoMediaPath { get; }
+        bool DebugMode { get; }
+
+        bool IsSmtpServerConfigured { get; }
+
+        /// <summary>
+        /// Gets a value indicating whether the runtime should enter Install level when the database is missing.
+        /// </summary>
+        /// <remarks>
+        /// <para>By default, when a database connection string is configured but it is not possible to
+        /// connect to the database, the runtime enters the BootFailed level. If this options is set to true,
+        /// it enters the Install level instead.</para>
+        /// <para>It is then up to the implementor, that is setting this value, to take over the installation
+        /// sequence.</para>
+        /// </remarks>
+        bool InstallMissingDatabase { get; }
+
+        /// <summary>
+        /// Gets a value indicating whether the runtime should enter Install level when the database is empty.
+        /// </summary>
+        /// <remarks>
+        /// <para>By default, when a database connection string is configured and it is possible to connect to
+        /// the database, but the database is empty, the runtime enters the BootFailed level. If this options
+        /// is set to true, it enters the Install level instead.</para>
+        /// <para>It is then up to the implementor, that is setting this value, to take over the installation
+        /// sequence.</para>
+        /// </remarks>
+        bool InstallEmptyDatabase { get; }
+        bool DisableElectionForSingleServer { get; }
+        string RegisterType { get; }
```

Comment:
Not sure where to find these comment.. These was introduced because ConfigurationManager was used directly - And as far as I can see without any comments

Reply:
Even in Constants.AppSettings.* These three was todo's in the xmldoc

Reply:
Hrm... i swear i saw a bunch of comments someplace for these but i can't find them either 🤔maybe i was just going a bit crazy looking at all the code review code :P